storybook/storybook-react/src/Dialog/Dialog.stories.tsx

```diff
@@ -0,0 +1,127 @@
+      <Dialog {...args} />
+    </div>
+  ),
+}
+
+export const ScrollContent: Story = {
+  args: {
+    open: true,
+    title: 'Privacyverklaring gemeente Amsterdam',
+    children: (
+      <>
+        <Heading size="level-4">Algemeen</Heading>
+        <Paragraph>
+          De gemeente Amsterdam verwerkt bij de uitvoering van haar taken en verplichtingen persoonsgegevens. De manier
+          waarop de gemeente Amsterdam om gaat met persoonsgegevens is vastgelegd in het stedelijk kader verwerken
+          persoonsgegevens.
+        </Paragraph>
+        <Paragraph>
+          Deze verklaring geeft aanvullende informatie over de omgang met persoonsgegevens door de gemeente Amsterdam en
+          over uw mogelijkheden tot het uitoefenen van uw rechten met betrekking tot persoonsgegevens.
+        </Paragraph>
+        <Paragraph>
+          Meer specifieke informatie over privacy en de verwerking van persoonsgegevens door de gemeente Amsterdam kunt
+          u op de hoofdpagina vinden.
+        </Paragraph>
+        <Paragraph>
+          Vanwege nieuwe wetgeving of andere ontwikkelingen, past de gemeente regelmatig haar processen aan. Dit kunnen
+          ook wijzigingen zijn in de wijze van het verwerken van persoonsgegevens. Wij raden u daarom aan om regelmatig
+          deze pagina te bekijken. Deze pagina wordt doorlopend geactualiseerd.
+        </Paragraph>
+        <Heading size="level-4">Geldende wet- en regelgeving en reikwijdte</Heading>
+        <Paragraph>
+          Vanaf 25 mei 2018 is de Algemene verordening gegevensbescherming (Avg) van toepassing op alle verwerkingen van
+          persoonsgegevens. Deze Europese wetgeving heeft directe werking in Nederland. Voor die zaken die nationaal
+          geregeld moeten worden, is de Uitvoeringswet Avg in Nederland aanvullend van toepassing. Deze wetteksten kunt
+          u vinden op de website van Autoriteit Persoonsgegevens.
+        </Paragraph>
+      </>
+    ),
+  },
+  render: (args) => (
+    <div style={{ minHeight: '100vh' }}>
+      <Dialog {...args} />
+    </div>
+  ),
+}
+
+export const ShowDialog: Story = {
+  args: {
+    id: 'showdialog',
+    actions: (
+      <>
+        <Button variant="tertiary" autoFocus onClick={(event) => event.currentTarget.closest('dialog')?.close()}>
+          Terug
+        </Button>
+        <Button type="submit">Verder</Button>
+      </>
+    ),
+  },
+  decorators: [
+    (Story) => (
+      <article>
+        <Button onClick={() => (document.querySelector('#showdialog') as HTMLDialogElement)?.showModal()}>
+          Open Dialog
+        </Button>
+        <Story />
+      </article>
+    ),
+  ],
+  parameters: {
+    backgrounds: {
+      disable: true,
+    },
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement)
+    await userEvent.click(canvas.getByRole('button'), { delay: 500 })
+    await userEvent.click(canvas.getByText('Sluiten'), { delay: 1000 })
+  },
```

Comment:
This currently doesn't really test anything, right? It just goes through the interaction? Maybe we can get rid of this for now, but it would be nice to figure out how we can use this for testing in a seperate ticket

Reply:
It tests if you can successfully open and close the dialog, something the unit test fails to do.

Reply:
But it doesn't really test that currently, right? Say the close button doesn't close the Dialog, would we get a notification now?

Reply:
Well just in storybook until I integrate jest or vitest, they recelty release a @storybook/test package that combines a lot of things. I'll comment it out until I have something to show for.

Reply:
Ok done, i'll add the ticket later after researching new development a bit.